tests/test_tools.py

```diff
@@ -22,6 +22,9 @@ def _run_tests(inputs, expected):
     config = get_user_info(args)
     assert config.get("username") == expected_username
     assert config.get("email") == expected_email
+    config = get_user_info(args, skip_config_creation=True)
+    assert config.get("username") == expected_username
+    assert config.get("email") == expected_email
```

Comment:
It looks like we can reuse this run_test function for most test cases for different `skip_config_creation` except when there're no inputs or files, so I added this here